cmake/iotjs.cmake

```diff
@@ -134,38 +145,28 @@ set(PLATFORM_SRC
 foreach(module ${IOTJS_NATIVE_MODULES})
   string(TOUPPER ${module} MODULE)
   # check if there is a native file for the module
-  set(BASE_MODULE_SRC ${IOTJS_SOURCE_DIR}/module/iotjs_module_${module}.c)
+  set(BASE_MODULE_SRC ${IOTJS_SOURCE_DIR}/modules/iotjs_module_${module}.c)
   if(EXISTS "${BASE_MODULE_SRC}")
     list(APPEND IOTJS_MODULE_SRC ${BASE_MODULE_SRC})
   endif()
 
-  # check if there is an extra platform support code for the module
-  # PLATFORM_DESCRIPTOR is <arch>-<os>  (eg.: x86_64-linux)
-  # ../platform/<arch>-<os>/iotjs_module_<module> ...
-  set(PLATFORM_MODULE_BASE ${PLATFORM_SRC}_${module}-${PLATFORM_DESCRIPTOR})
-  set(PLATFORM_MODULE_SRC ${PLATFORM_MODULE_BASE}.c)
-  if(VERBOSE)
-    message("Checking platform file: ${PLATFORM_MODULE_SRC}")
-  endif()
-  if(EXISTS "${PLATFORM_MODULE_SRC}")
-    list(APPEND IOTJS_MODULE_SRC ${PLATFORM_MODULE_SRC})
-    list(APPEND IOTJS_PLATFORM_SUPPORT ${MODULE})
+  # first, check if there is the module in <os>/<os>-<board>
+  # ../platform/<os>/<os>-board>iotjs_module_<module> ...
```

Comment:
Cannot interpret this line of comment. (Perhaps a leftover from the original variant of the cmake file?)